portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/upgrades/DBStoreUpgrade.testcase

```diff
@@ -0,0 +1,44 @@
+@component-name = "portal-database-upgrade-framework"
+definition {
+
+	property custom.properties = "dl.store.impl=com.liferay.portal.store.db.DBStore";
+	property portal.release = "true";
+	property portal.upgrades = "true";
+	property portal.upstream = "true";
+	property test.assert.warning.exceptions = "false";
```

Comment:
Is it better to set this to false or ignore the specific error in test-ignorable-error-lines.xml? [LPS-187865](https://issues.liferay.com/browse/LPS-187865)

Reply:
@susanchen3 This is only affecting DB2? If so, just restrict the database.types to not run on DB2 for now.

Reply:
@vicnate5 done, removed DB2